server/service/src/sync/translations/location.rs

```diff
@@ -76,6 +77,7 @@ impl SyncTranslation for LocationTranslation {
             code,
             on_hold,
             store_id,
+            cold_storage_type_id: _,
```

Comment:
```suggestion
            cold_storage_type_id: _, // TODO: Translate cold_storage_type_id id from `location_type_id`
```
I prefer to add a comment in the code to make it clear this will need to be resolved in future PR.